### PR TITLE
sort the list and remove duplicate op

### PR DESCRIPTION
**Pull request by yuchengliu1**
# OPS in `xpu_functions.yaml` but not in `_xpu_computation_op_list`:
    "complex.out",
    "zero_",
    "rsub",
    "random",
    "nn.functional.dropout",
    "to_copy",
    "set_",
    "_copy_from_and_resize",
    "_copy_from",
    "nn.functional.layer_norm",
    "index",
    "max",
    "min",
    "mean",
    "argmax",
    "_local_scalar_dense",# or scalar_tensor
    "col2im",
    "batch_norm_stats",
    "batch_norm_elemt",
    "batch_norm_update_stats",
    "batch_norm_backward",
    "upsample_bicubic2d",
    "upsample_bilinear2d_backward",
    "_embedding_bag",
    "is_set_to",
    "reflection_pad1d",# and its variants
    "replication_pad1d",# and its variants
    "_amp_foreach_non_finite_check_and_unscale_",
    "_amp_update_scale_",
    "ceil",
    "upsample_linear1d",
    "_weight_norm_interface",
    "range",
    "_efficientzerotensor",
    "_index_put_impl_",

# OPS in `_xpu_computation_op_list` but not in `xpu_functions.yaml`:
    "zeros",
    "zeros_like",
    "copy",
    "index_put",
    "nn.functional.pad",
    "reshape",
    "nanmean",

### CODE REVIEW

**Comment by majing921201** [comment]
Which is duplicate op we removed in this pr ?

**Comment by yuchengliu1** [comment]
> Which is duplicate op we removed in this pr ?

Although this PR removed duplicate ops by sort it, this PR not focus on duplicate ops. Do you need the list of duplicate ops?

**Comment by majing921201** [comment]
> > Which is duplicate op we removed in this pr ?
> 
> Although this PR removed duplicate ops by sort it, this PR not focus on duplicate ops. Do you need the list of duplicate ops?

thanks, no need the list, I am just curious, because I didn't find duplicate op, maybe I missed.
